AuroraAssetEditor/Classes/XboxAssetDownloader.cs

```diff
@@ -177,12 +177,12 @@ public enum XboxAssetType {
                 }
             }
             titleInfo.AssetsInfo = ret.ToArray();
-            XboxAssetDownloader.SendStatusChanged("Finished parsing Title/Asset info...");
+            XboxAssetDownloader.SendStatusChanged("Finished fetching assets!");
```

Comment:
This should probably be "Finished parsing asset information..."

Reply:
Fair enough, I can change this as well as the others you suggested.